FireboltNETSDK/Client/FireboltCommand.cs

```diff
@@ -178,15 +178,19 @@ private DbDataReader CreateDbDataReader(QueryResult? queryResult)
             {
                 commandText = ValidateSetCommand(commandText.Remove(0, 4).Trim());
                 SetParamList.Add(commandText);
-                try
+                if (Connection != null)
```

Comment:
Do we need this check? We already verify Connection is not null in https://github.com/firebolt-db/firebolt-net-sdk/pull/106/files#diff-9f5e4d8a0218e3f437d1707871b746709b804521e38023b2328a94ffb378452aR168

Reply:
Good point, I did this for type checker